.github/workflows/docs-and-linting.yml

```diff
@@ -28,17 +29,19 @@ jobs:
         with:
           go-version: ${{ matrix.go != 'go.mod' && matrix.go || null }}
           go-version-file: ${{ matrix.go == 'go.mod' && 'go/src/github.com/opencontainers/image-spec/go.mod' || null }}
```

Comment:
If we _do_ want to test both `go.mod` versions (and maybe even if we don't?) this could switch to something like `endsWith(matrix.go, 'go.mod')` and then use the value of `matrix.go` as the file directly (even better with https://github.com/opencontainers/image-spec/pull/1264).